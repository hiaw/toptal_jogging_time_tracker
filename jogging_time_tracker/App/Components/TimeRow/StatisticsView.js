/* @flow*/
import React from 'react'
import { ScrollView } from 'react-native'

import StatisticsRecordSpeed from './StatisticsRecordSpeed.js'
import StatisticsWeeklyComparison from './StatisticsWeekComparison.js'
import StatisticsOthers from './StatisticsOthers.js'

import styles from './Styles/StatisticsView.style.js'

const StatisticView = props => {
  return (
    <ScrollView contentContainerStyle={styles.container}>
      <StatisticsRecordSpeed {...props} />

      <StatisticsWeeklyComparison {...props} />

      <StatisticsOthers {...props} />
    </ScrollView>
  )
}

export default StatisticView
