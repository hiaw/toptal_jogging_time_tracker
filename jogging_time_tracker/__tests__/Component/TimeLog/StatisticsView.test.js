import 'react-native'
import React from 'react'

import StatisticsView from '../../../App/Components/TimeRow/StatisticsView.js'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

const noop = () => {}

const data = []

it('renders empty array', () => {
  const tree = renderer.create(<StatisticsView data={data} />).toJSON()
  expect(tree).toMatchSnapshot()
})

Date.now = jest.fn(() => 1487076708000) //14.02.2017

const newData = [
  { date: '2017-02-01T08:47:26.504Z', distance: 840.11, duration: 30.84 },
  { date: '2017-02-09T08:46:59.508Z', distance: 5488.61, duration: 781.28 },
  { date: '2017-02-13T08:47:15.508Z', distance: 5100.16, duration: 632.35 },
]

it('renders empty array', () => {
  const tree = renderer.create(<StatisticsView data={newData} />).toJSON()
  expect(tree).toMatchSnapshot()
})
