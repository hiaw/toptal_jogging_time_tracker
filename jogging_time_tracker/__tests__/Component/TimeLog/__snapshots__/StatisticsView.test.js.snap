// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders empty array 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "paddingBottom": 20,
      "paddingTop": 60,
    }
  }
>
  <View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderColor": "#e1e8ee",
            "borderWidth": 1,
            "margin": 15,
            "marginBottom": 0,
            "padding": 15,
            "shadowColor": "rgba(0,0,0, .2)",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 1,
            "shadowRadius": 1,
          },
          undefined,
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
            undefined,
          ]
        }
      >
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "#43484d",
                    "fontSize": 17.5,
                    "fontWeight": "bold",
                    "marginBottom": 15,
                    "textAlign": "center",
                  },
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            Record Speed
          </Text>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#e1e8ee",
                  "height": 1,
                },
                Array [
                  Object {
                    "marginBottom": 15,
                  },
                  undefined,
                ],
              ]
            }
          />
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            />
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Week
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Month
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Year
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Fastest
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Slowest
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderColor": "#e1e8ee",
            "borderWidth": 1,
            "margin": 15,
            "marginBottom": 0,
            "padding": 15,
            "shadowColor": "rgba(0,0,0, .2)",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 1,
            "shadowRadius": 1,
          },
          undefined,
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
            undefined,
          ]
        }
      >
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "#43484d",
                    "fontSize": 17.5,
                    "fontWeight": "bold",
                    "marginBottom": 15,
                    "textAlign": "center",
                  },
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            Last Week Comparison
          </Text>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#e1e8ee",
                  "height": 1,
                },
                Array [
                  Object {
                    "marginBottom": 15,
                  },
                  undefined,
                ],
              ]
            }
          />
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            />
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Distance
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Duration
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Speed
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Last Week
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0 m
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0:00:00
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              NaN km/h
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Week
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0 m
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0:00:00
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              NaN km/h
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderColor": "#e1e8ee",
              "borderWidth": 1,
              "margin": 15,
              "marginBottom": 0,
              "padding": 15,
              "shadowColor": "rgba(0,0,0, .2)",
              "shadowOffset": Object {
                "height": 0,
                "width": 0,
              },
              "shadowOpacity": 1,
              "shadowRadius": 1,
            },
            undefined,
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
              undefined,
            ]
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Array [
                    Object {
                      "color": "#43484d",
                      "fontSize": 17.5,
                      "fontWeight": "bold",
                      "marginBottom": 15,
                      "textAlign": "center",
                    },
                    undefined,
                    undefined,
                    undefined,
                  ],
                ]
              }
            >
              Greatest Distance Day
            </Text>
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#e1e8ee",
                    "height": 1,
                  },
                  Array [
                    Object {
                      "marginBottom": 15,
                    },
                    undefined,
                  ],
                ]
              }
            />
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Not Found
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0.00m
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderColor": "#e1e8ee",
              "borderWidth": 1,
              "margin": 15,
              "marginBottom": 0,
              "padding": 15,
              "shadowColor": "rgba(0,0,0, .2)",
              "shadowOffset": Object {
                "height": 0,
                "width": 0,
              },
              "shadowOpacity": 1,
              "shadowRadius": 1,
            },
            undefined,
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
              undefined,
            ]
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Array [
                    Object {
                      "color": "#43484d",
                      "fontSize": 17.5,
                      "fontWeight": "bold",
                      "marginBottom": 15,
                      "textAlign": "center",
                    },
                    undefined,
                    undefined,
                    undefined,
                  ],
                ]
              }
            >
              Cumulative
            </Text>
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#e1e8ee",
                    "height": 1,
                  },
                  Array [
                    Object {
                      "marginBottom": 15,
                    },
                    undefined,
                  ],
                ]
              }
            />
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              % Distance To Moon:
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Distance: 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0 m
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Time: 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0:00:00
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;

exports[`renders empty array 2`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "paddingBottom": 20,
      "paddingTop": 60,
    }
  }
>
  <View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderColor": "#e1e8ee",
            "borderWidth": 1,
            "margin": 15,
            "marginBottom": 0,
            "padding": 15,
            "shadowColor": "rgba(0,0,0, .2)",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 1,
            "shadowRadius": 1,
          },
          undefined,
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
            undefined,
          ]
        }
      >
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "#43484d",
                    "fontSize": 17.5,
                    "fontWeight": "bold",
                    "marginBottom": 15,
                    "textAlign": "center",
                  },
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            Record Speed
          </Text>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#e1e8ee",
                  "height": 1,
                },
                Array [
                  Object {
                    "marginBottom": 15,
                  },
                  undefined,
                ],
              ]
            }
          />
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            />
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Week
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Month
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Year
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Fastest
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              27.240920881971466
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              27.240920881971466
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Slowest
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              27.240920881971466
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              27.240920881971466
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "borderColor": "#e1e8ee",
            "borderWidth": 1,
            "margin": 15,
            "marginBottom": 0,
            "padding": 15,
            "shadowColor": "rgba(0,0,0, .2)",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 1,
            "shadowRadius": 1,
          },
          undefined,
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
            undefined,
          ]
        }
      >
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {},
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Array [
                  Object {
                    "color": "#43484d",
                    "fontSize": 17.5,
                    "fontWeight": "bold",
                    "marginBottom": 15,
                    "textAlign": "center",
                  },
                  undefined,
                  undefined,
                  undefined,
                ],
              ]
            }
          >
            Last Week Comparison
          </Text>
          <View
            style={
              Array [
                Object {
                  "backgroundColor": "#e1e8ee",
                  "height": 1,
                },
                Array [
                  Object {
                    "marginBottom": 15,
                  },
                  undefined,
                ],
              ]
            }
          />
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
            }
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            />
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Distance
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Duration
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Speed
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Last Week
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              5.1 km
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0:10:32.4
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              29 km/h
            </Text>
          </View>
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              This Week
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0 m
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0:00:00
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              NaN km/h
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderColor": "#e1e8ee",
              "borderWidth": 1,
              "margin": 15,
              "marginBottom": 0,
              "padding": 15,
              "shadowColor": "rgba(0,0,0, .2)",
              "shadowOffset": Object {
                "height": 0,
                "width": 0,
              },
              "shadowOpacity": 1,
              "shadowRadius": 1,
            },
            undefined,
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
              undefined,
            ]
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Array [
                    Object {
                      "color": "#43484d",
                      "fontSize": 17.5,
                      "fontWeight": "bold",
                      "marginBottom": 15,
                      "textAlign": "center",
                    },
                    undefined,
                    undefined,
                    undefined,
                  ],
                ]
              }
            >
              Greatest Distance Day
            </Text>
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#e1e8ee",
                    "height": 1,
                  },
                  Array [
                    Object {
                      "marginBottom": 15,
                    },
                    undefined,
                  ],
                ]
              }
            />
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Not Found
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0.00m
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "white",
              "borderColor": "#e1e8ee",
              "borderWidth": 1,
              "margin": 15,
              "marginBottom": 0,
              "padding": 15,
              "shadowColor": "rgba(0,0,0, .2)",
              "shadowOffset": Object {
                "height": 0,
                "width": 0,
              },
              "shadowOpacity": 1,
              "shadowRadius": 1,
            },
            undefined,
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
              undefined,
            ]
          }
        >
          <View>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Array [
                    Object {
                      "color": "#43484d",
                      "fontSize": 17.5,
                      "fontWeight": "bold",
                      "marginBottom": 15,
                      "textAlign": "center",
                    },
                    undefined,
                    undefined,
                    undefined,
                  ],
                ]
              }
            >
              Cumulative
            </Text>
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#e1e8ee",
                    "height": 1,
                  },
                  Array [
                    Object {
                      "marginBottom": 15,
                    },
                    undefined,
                  ],
                ]
              }
            />
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              % Distance To Moon:
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0.003
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Distance: 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              11.43 km
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              Time: 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              disabled={false}
              ellipsizeMode="tail"
            >
              0:24:04.5
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
